db/schema.rb

```diff
@@ -10,11 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_03_28_191652) do
-
+ActiveRecord::Schema.define(version: 2020_03_28_134851) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+<<<<<<< HEAD
```

Comment:
@kopchickm There seems to be a merge conflict

Reply:
Thanks, fixed!